src/agent/coverage/src/allowlist.rs

```diff
@@ -144,7 +144,13 @@ fn glob_to_regex(expr: &str) -> Result<Regex> {
     // Anchor to line start and end.
     let expr = format!("^{expr}$");
 
-    Ok(Regex::new(&expr)?)
+    let mut rb = RegexBuilder::new(&expr);
+
+    if cfg!(windows) {
+        rb.case_insensitive(true);
```

Comment:
The allowlist matching is unit-tested. I suggest adding at least one test decorated with `[cfg(target_os = "windows")]` which exercises the new matching logic. Maybe also a Linux-only test that demonstrates case-sensitivity on that platform.

Reply:
That unit test caught a bug! Setting the case_inensitive flag in the RegexBuilder didn't get passed along when we convert from Regex to RegexSet in a different part of the code.